### PR TITLE
refactor(ui-toolkit): move healthbar to ui-toolkit

### LINKED ISSUE #2791: Move/refactor health-bar to ui-toolkit
- [ ] Move health-bar to ui-toolkit
- [ ] Make health-bar usable as a generic presentational component

### PR DESCRIPTION
**Pull request by ciaran-**
# Related issues 🔗

Closes #2791

# Description ℹ️

Moves the healthbar from the lp-dashboard project to ui-toolkit so we can re-use it.

# Demo 📺

Screenshot 2023-03-21 at 17 19 53

# Technical 👨‍🔧

Moves the ‘healthbar’ component to ui-toolkit and includes some other changes to make that possible - using the ui-toolkit `Tooltip` and `Indicator` components replacing the duplicate lp-dashboard-specific ones.

Also makes some changes to how it looks/behaves to address: #1816

### CODE REVIEW

File: apps/liquidity-provision-dashboard/.env [hunk 1]
```diff
@@ -19,7 +19,7 @@ NX_DEPLOY_URL=$DEPLOY_URL
 NX_DEPLOY_PRIME_URL=$DEPLOY_PRIME_URL
 NX_VEGA_CONFIG_URL=https://raw.githubusercontent.com/vegaprotocol/networks-internal/main/fairground/vegawallet-fairground.toml
 NX_VEGA_ENV = 'TESTNET'
-NX_VEGA_URL="https://api.n11.testnet.vega.xyz/graphql"
+NX_VEGA_URL="https://api.n08.testnet.vega.xyz/graphql"
```
Comment (dexturr) [inline apps/liquidity-provision-dashboard/.env:22]:
```suggestion
NX_VEGA_URL="https://api.n11.testnet.vega.xyz/graphql"
```

File: libs/ui-toolkit/src/components/healthbar/healthbar.tsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
+import classNames from 'classnames';
+import * as Schema from '@vegaprotocol/types';
```
Comment (dexturr) [inline libs/ui-toolkit/src/components/healthbar/healthbar.tsx:2]: Toolkit package shouldn't care about what types are/are not in Vega conceptually.

Comment (dexturr) [reply libs/ui-toolkit/src/components/healthbar/healthbar.tsx:2]: Suggest externalising this dependency by taking an intent prop

Comment (ciaran-) [reply libs/ui-toolkit/src/components/healthbar/healthbar.tsx]: Made this change

**Review by dexturr — APPROVED** [review]

**Pull request merged**